chapters/ranker.rst

```diff
@@ -0,0 +1,155 @@
+==========================================
+A Guide to Jina Ranker
+==========================================
+
+.. meta::
+   :description: A guide on Jina Ranker
+   :keywords: Jina, Ranker
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+--------------------
+
+In a searching system, query accuracy is one of the most important aspects that users will concern. In most cases, we expect well-ordered query results to be presented.
+For one search request, we may have multiple matches and we want to have the most relevant match or probably top-K matches presented to users. On the other hand, we may also want to aggregate the top-K chunks to top-K matches.
+
+
+Before you start
+-------------------
+
+We expect you have a clean Python 3.7/3.8/3.9 (virtual) environment.
+Install Jina on your machine:
+
+.. highlight:: bash
+.. code-block:: bash
+
+    pip install -U jina
+
+
+Overview
+-----------------
+
+To achieve our objectives, Jina has a family of :term:`Executor` named :term:`Ranker`.
+Jina :term:`Ranker` inherits either the :class:`Match2DocRanker` or the :class:`Chunk2DocRanker`.
+For the :class:`Match2DocRanker`, it re-scores the matches with different mechanisms. For the :class:`Chunk2DocRanker`, it aggegrates the chunks to documents.
+
+
+Match2DocRanker
+^^^^^^^^^^^^^^^^
+
+:class:`Match2DocRanker` re-scores the matches for a document. This Ranker is only responsible for calculating new scores and not for the actual sorting. The sorting is handled in the respective :class:`Matches2DocRankDriver`.
```

Comment:
**D**ocument

Reply:
Check the whole document for this issue.

Reply:
✅